.appveyor.yml

```diff
@@ -4,6 +4,8 @@ environment:
   global:
     DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
     DOTNET_CLI_TELEMETRY_OPTOUT: 1
+  ZEUS_HOOK_BASE:
+    secure: dcqtt6sxxBV0tDkXmPZOy/mS9w6lpsiR4cz0HHfryT5/NHNfboPH5/VaOFvOXnh2gD3ftUutQ6h0GDUWT9HrKhZoqq2kqv20pevaRU0MOBTCJ92HiW6vSSkT2niMA/ZvWFtQGO2tdPURILtZ99aeZ7sirOjrSwRRoy/bZpo6Uzg=
```

Comment:
Hmm, looks like ZEUS_HOOK_BASE was not added to the repo settings in Appveyor

Reply:
I added there and then realized there's no need to add in both. It's either added via the UI or via the config with `secure`. 
I've opted to add it here only then.